specification/electronic-prescription-service-api.yaml

```diff
@@ -557,13 +557,20 @@ paths:
       summary: Mark a prescription as dispensed
       description: |
         ## Overview
+        ### Dispense Notification
         Use this endpoint to send a dispense-notification message to EPS.
 
         The effect of calling this endpoint depends on the type of the message, which is determined by the `MessageHeader` resource.
         To mark a prescription as partially or fully dispensed, send a [dispense-notification message](https://simplifier.net/guide/NHSDigital-Medicines/Home/FHIRAssets/AllAssets/MessageDefinition-duplicate-2/dispense-notification.guide.md).
 
         Note that the `#dispense-notification` in the URL has been added to distinguish this from the other operations which
         can be performed using the [`$process-message` endpoint](https://simplifier.net/guide/NHSDigital/Home/FHIRAssets/AllAssets/OperationDefinition/process-message.guide.md). It is not required in requests to the API.
+        
+        ### Amend Dispense Notification
+        The same endpoint can be used to amend a previous dispense notification. The same message structure is used with the addition of a `replacementOf` extension in the MessageHeader,
+        referencing the identifier of the previous dispense notifcation to be amended. 
+        
+        More information can be found on the fhir [documentation](https://simplifier.net/guide/NHSDigital/Home/FHIRAssets/AllAssets/Profiles/NHSDigital-MessageHeader.guide.md).
```

Comment:
do we want to specify that only the latest notification can be amended and that do amend others you need to withdraw first?